api/src/data/entities/dictionary.clj

```diff
@@ -52,16 +55,41 @@
   (db/update! dictionary-combined-db (:key item) (dissoc item :key)))
 
 (defn list-dict-files []
-  (filter #(.isFile ^File %) (file-seq (io/file (or (System/getenv "DICT_PATH") "grammar/dictionary")))))
+  (->> (file-seq (io/file (or (System/getenv "DICT_PATH") "grammar/dictionary")))
+       (filter #(.isFile ^File %))
+       (filter #(str/ends-with? (.getName %) "yaml"))))
+
+(defn create-multilang-dict-item [{::morphology/keys [key language gender pos sense tenses inflections]}]
+  (db/write! dictionary-multilang-db (utils/gen-uuid) {:key         key
+                                                       :language    language
+                                                       :gender      gender
+                                                       :pos         pos
+                                                       :sense       sense
+                                                       :tenses      tenses
+                                                       :inflections inflections}))
+
+(defn get-multidict-items [key]
+  (db/read! dictionary-multilang-db key))
+
+(defn search-multilang-dict
+  ([key]
+   (db/scan! dictionary-multilang-db {:key key}))
+  ([key pos senses]
+   (db/scan! dictionary-multilang-db {:key key :pos pos :senses senses})))
+
+
+
+(defn list-multilang-dict [limit]
+  (db/list! dictionary-multilang-db limit))
 
 (defn initialize []
-  (doseq [f (list-dict-files)]
-    (let [{:keys [phrases partOfSpeech name]} (yaml/parse-string (slurp f))
-          filename (utils/get-name f)]
-      (when-not (get-dictionary-item filename)
-        (create-dictionary-item
-          {:key          filename
-           :name         (or name filename)
-           :phrases      phrases
-           :partOfSpeech (when (some? partOfSpeech)
-                           (keyword partOfSpeech))})))))
+  (db/write! reader-flags-db :English  :YES)
```

Comment:
At `data.entities.dictionary` we have
```
(defn get-default-flags []
  {:English  :YES
   :Estonian :NO
   :German   :NO
   :Latvian  :NO
   :Russian  :NO})
```
Looks like similar to what you do here.  Any chance of reuse/duplication?